src/openqa_client/client.py

```diff
@@ -377,3 +377,33 @@ def get_jobs(
             # there as it only considers the jobs queried.
             jobdicts = self.find_clones(jobdicts)
         return jobdicts
+
+    def get_latest_build(
+        self,
+        group_id: int,
+        all_passed: bool = True,
+        sorted_key: Callable = float,
+        default_return: str = "0",
+    ) -> str:
+        """Identify latest build number in target Job Group
+
+        Args:
+            group_id (int): Job Group ID
+            all_passed (bool, optional): Controls whether just
+                last one will be selected or last with all jobs passed. Defaults to True.
+            sorted_key (Callable, optional): To judge about latest build we need to order them.
+                This is callback which will be used in sorted function to sort builds.
+                Defaults to float.
+            default_return (str, optional): In case all available builds have not acceptable format
+                or there is no builds in this job group this value identofy what to return. Defaults to "0".
```

Comment:
```suggestion
            default_return (str, optional): In case all available builds have no acceptable format
                or there are no builds in this job group this value specifies what to return. Defaults to "0".
```

Reply:
done